docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/GracefulShutdownStrategy.java

```diff
@@ -18,11 +18,16 @@
     private static final Logger log = LoggerFactory.getLogger(GracefulShutdownStrategy.class);
 
     @Override
-    public void shutdown(DockerCompose dockerCompose, Docker docker) throws IOException, InterruptedException {
-        log.debug("Killing docker-compose cluster");
-        dockerCompose.down();
+    public void stop(DockerCompose dockerCompose) throws IOException, InterruptedException {
+        log.debug("Stopping docker-compose cluster");
+        dockerCompose.stop();
         dockerCompose.kill();
-        dockerCompose.rm();
```

Comment:
Was the `.rm()` intentionally deleted?

Reply:
It is redundant, `.down()` implies `.rm()`.

Reply:
The act of removing containers has been separated in the `shutdown` method.